doc/destructors.rst

```diff
@@ -522,7 +522,7 @@ a form of copy elision.
 
 To see how and when we can do that, think about this question: In `dest = src` when
 do we really have to *materialize* the full copy? - Only if `dest` or `src` are mutated
-afterwards. If `dest` is a local variable that is simple to analyse. And if `src` is a
+afterward. If `dest` is a local variable that is simple to analyze. And if `src` is a
```

Comment:
"afterwards" and "afterward" are grammatically and semantically equivalent - their usage depends on regional preference.

Reply:
I've researched the issue too and it looks like you're right. So, I guess I could leave them as they are...